### PR TITLE
WX-864 Add final workflow log directory config for Cromwell

### PR DESCRIPTION
**Pull request by kraefrei**
This change adds a config for Cromwell in CBAS to specify a final directory to put execution logs, particularly for blob storage. This change works with a change in CBAS to supply this value to WES but also requires an outstanding change with the WES API to ensure the workflow option is used for the submission

### CODE REVIEW

**Comment by cjllanwarne** [comment]
Pedantic comment, but it might help a future ticket searcher: this change is for cromwell config rather than CBAS

**Review by cjllanwarne — APPROVED** [review]

**Comment by jgainerdewar** [comment]
Can you also remove the Cromwell config that attempts to define the location for these logs?

File: cromwell-helm/templates/cbas-api.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ data:
       cromwell:
         baseUri: "http://{{ include "app.fullname" . }}-cromwell-svc:8000"
         healthUri: "http://{{ include "app.fullname" . }}-cromwell-svc:8000/engine/v1/status"
+        finalWorkflowLogDir: "https://{{ .Values.persistence.storageAccount }}.blob.core.windows.net/{{ .Values.persistence.blobContainer }}/workspace-services/cbas/{{ .Values.persistence.leoAppInstanceName }}/cromwell-workflow-logs"
```
Comment (cjllanwarne) [inline cromwell-helm/templates/cbas-api.yaml:31]: Isn't this the GCP version of the app? If so it probably doesn't want this value

**Review by salonishah11 — APPROVED** [review]

**Review by cjllanwarne — CHANGES_REQUESTED** [review]

File: cromwell-helm/templates/cbas-api.yaml (hunk 1 above)
Comment (kraefrei) [reply cromwell-helm/templates/cbas-api.yaml]: I had not realized! Thank makes sense though, I have now removed it

**Review by cjllanwarne — APPROVED** [review]

**Pull request merged**